src/output-eve-null.c

```diff
@@ -0,0 +1,87 @@
+/* vi: set et ts=4: */
```

Comment:
nit: accidental conf?

Reply:
Not intentional and I can see that there are 3 other source modules that have this as well.

I don't mind removing it from all of them.

Thoughts?

Reply:
Removing vi formatting directives in followup pr